js/app/src/Login.stories.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+	import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+	import Login from "./Login.svelte";
+</script>
+
+<Meta
+	title="Login Page"
+	component={Login}
+	argTypes={{
+		auth_message: {
+			control: "text",
+			description: "The optional auth message to display",
+			name: "label",
+			value: "Gradio Button"
+		}
+	}}
+/>
+
+<Template let:args>
+	<Login {...args} />
+</Template>
+
+<Story name="Login page without auth message" args={{ auth_message: null }} />
```

Comment:
Was super easy to set this up! However, the theme is not being applied properly. The page is using a seriffed-font. Any idea why @hannahblair @pngwn ?

<img width="599" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/4605df56-3309-416b-a3e4-0f971d89560d">

Reply:
I'm not seeing this behaviour. The font looks correct to me.

Reply:
It fixed itself!!